fictadvisor-web/src/components/pages/admin/disciplines-admin-page/components/disciplines-table/components/table-actions/TableActions.styles.ts

```diff
@@ -0,0 +1,12 @@
+import { SxProps, Theme } from '@mui/material/styles';
+
+export const tableColumn: SxProps<Theme> = {
+  flexDirection: 'row',
```

Comment:
same here

Reply:
we don`t need a display: flex here, as there is a Stack from mui, which have display: flex by default